doxia-core/src/main/java/org/apache/maven/doxia/sink/impl/RandomAccessSink.java

```diff
@@ -27,19 +27,21 @@
 import java.util.List;
 
 import org.apache.maven.doxia.sink.Sink;
-import org.apache.maven.doxia.sink.SinkEventAttributes;
 import org.apache.maven.doxia.sink.SinkFactory;
 
 /**
  * The RandomAccessSink provides the ability to create a {@link Sink} with hooks.
  * A page can be prepared by first creating its structure and specifying the positions of these hooks.
  * After specifying the structure, the page can be filled with content from one or more models.
  * These hooks can prevent you to have to loop over the model multiple times to build the page as desired.
- *
+ * Consider using the {@link BufferingSinkProxyFactory} instead which allows to buffer on the (higher) Sink API level
```

Comment:
Should we deprecate in favour of `SinkWrapper`?

Reply:
Same as `PipelineSink`

Reply:
Ok, deprecated it in https://github.com/apache/maven-doxia/pull/184/commits/2695cb63c7b638f5a01c961f320c5a785eb192b3.